core/src/main/java/dev/streamx/cli/ingestion/publish/payload/PayloadResolver.java

```diff
@@ -131,49 +107,17 @@ private void replaceValues(DocumentContext documentContext, List<ValueArguments>
     }
   }
 
-  private static JsonNode extractReplacement(ValueArguments valueArgument,
-      Pair<JsonPath, String> extract) {
-    JsonPath jsonPath = extract.getKey();
-    String value = extract.getValue();
-
+  private JsonNode extractPayloadFragment(PayloadArgument valueArgument, String value)
```

Comment:
Why do we need value parameter here? The same data is available in `valueArgument` param.

Reply:
nope - `valueArgument` is raw value before splitting to jsonPath and replaced value. Argument `value` is already splitted. 
e.g. for option `-s content.bytes=file://path` results in `valueArgument.value` = `content.bytes=file://path` and argument passed to method `value` = `file://path`